### PR TITLE
Build site on S3 (tidy PR this time)

### PR DESCRIPTION
**Pull request by edunham**
@alexcrichton has already enabled Travis for the repo. 

I tested all this in my fork and got http://www-rust-lang-org.s3-website-us-west-1.amazonaws.com/ working. This will do the exact same thing (only change to the `.travis.yml` is specifying the repo owned by rust-lang rather than by me). 

This PR will not affect the behavior of the current (github pages) site at all, since it's to master.

### CODE REVIEW

**Comment by edunham** [comment]
Helps fix https://github.com/rust-lang/rust-www/issues/148 .

**Comment by steveklabnik** [comment]
I have one tiny comment, let me send you a PR :)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+- 2.1
+install: gem install jekyll
```
Comment (steveklabnik) [inline .travis.yml]: this line shouldn't need to exist, as we already have a Gemfile, and travis runs `bundle install -j 3` automatically if you have one.

**Comment by brson** [comment]
Looks good.

**Pull request merged**